source/linux/Foundational_Components/Kernel/Kernel_Drivers/Audio.rst

```diff
@@ -584,24 +578,145 @@ Board-specific instructions
     .. rubric:: User space
        :name: user-space-9
 
-.. _hdmi_audio:
-    The hardware defaults are correct for audio playback, the routing is OK
-    and the volume is 'adequate' but in case the volume is not correct:
+.. ifconfig:: CONFIG_part_variant not in ('AM62DX')
+
+	.. _hdmi_audio:
```

Comment:
This seems to be an unused label. It should probably be removed. It also doesn't help that it's under an ifconfig section. See https://github.com/TexasInstruments/processor-sdk-doc/blob/master/CONTRIBUTING.md#behavior for more information.